### PR TITLE
Pavel dimitrov fixing login and register

### PR DESCRIPTION
**Pull request by 2HEADSs**
Remove "createdEvent" field from user model
Fix return value when make request for register and login for both User and Organization

### CODE REVIEW

File: src/services/organizationService.js [hunk 1]
```diff
@@ -49,19 +49,27 @@ TODO:
 */
 
 function createToken(organization) {
+  //TODO: What payload will contain!
   const payload = {
     _id: organization.id,
     email: organization.email,
+    email: organization.email,
```
Comment (Getsov) [inline src/services/organizationService.js:56]: Is the email duplicated?

File: src/services/userService.js [hunk 2]
```diff
@@ -38,16 +38,23 @@ async function loginUser(email, password) {
...
 
 function createToken(user) {
-  //TODO: ADD EMAIL IN RETURN VALUE
+  //TODO: What payload will contain!
   const payload = {
     _id: user.id,
     firstName: user.firstName,
     lastName: user.lastName,
+    role: user.role,
+    city: user.city,
```
Comment (Getsov) [inline src/services/userService.js:47]: All models city -> region

File: src/services/userService.js [hunk 3]
```diff
@@ -38,16 +38,23 @@ async function loginUser(email, password) {
...
   };
   return {
     _id: user.id,
+    email: user.email,
     firstName: user.firstName,
     lastName: user.lastName,
+    role: user.role,
+    city: user.city,
```
Comment (Getsov) [inline src/services/userService.js:55]: Same here

Comment (2HEADSs) [reply src/services/userService.js]: Changed and published in the same branch.

File: src/services/userService.js (hunk 2 above)
Comment (2HEADSs) [reply src/services/userService.js]: Changed and published in the same branch.

File: src/services/organizationService.js (hunk 1 above)
Comment (2HEADSs) [reply src/services/organizationService.js:56]: Yes, already fixed and published in the same branch.

**Review by Getsov — APPROVED** [review]

**Pull request merged**